packages/scss/src/components/_container.scss

```diff
@@ -1,21 +1,21 @@
 .container {
 	margin: 0 auto;
 	padding: 0 2rem;
-	width: $xl - 40px;
+	width: _get($luTheme, "breakpoints.xl") - 40px;
```

Comment:
I set a -40px (which is a totaly arbitrary value) to have a container alway less wide than the view port, under each breakpoint. 

So it's only a way to define the container width relatively from our breakpoints.